assets/scss/layouts/_header.scss

```diff
@@ -168,8 +168,7 @@ button#doks-versions {
 
   .doks-search {
     max-width: 20rem;
-    margin-top: 0.125rem;
-    margin-bottom: 0.125rem;
+    margin: 0 auto;
```

Comment:
How does this change affect search on inside pages, like https://docs.blubracket.com/intro/key-workflows/ ?

Reply:
It centers it on those pages as well.
<img width="1419" alt="centeredsearch" src="https://user-images.githubusercontent.com/33102078/157172423-8868c805-a127-462d-8b88-63562166d67d.png">